code/modules/mining/equipment_locker.dm

```diff
@@ -597,7 +597,7 @@
 	..()
 
 /obj/machinery/mineral/equipment_vendor/proc/RedeemVoucher(obj/item/mining_voucher/voucher, mob/redeemer)
-	var/items = list("Kinetic Accelerator", "Survival Capsule and Explorer's Webbing", "Resonator Kit", "Minebot Kit", "Crusher Kit", "Mining Conscription Kit", "Advanced Scanner")
+	var/items = list("Lazarus Kit", "Kinetic Accelerator", "Survival Capsule and Explorer's Webbing", "Resonator Kit", "Minebot Kit", "Crusher Kit", "Mining Conscription Kit", "Advanced Scanner")
```

Comment:
This shouldn't really go as the first item on the list, considering it's less important than the other ones. I'd put it at the end.

Reply:
oh right its the click and enter meme. eh but sure